questionary/version.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.1"
+__version__ = "2.1.0"
```

Comment:
```suggestion
__version__ = "2.0.1"
```

The version will be bumped when we make a new release.